mount.c

```diff
@@ -90,6 +90,22 @@ void pv_mount_umount(void)
 	}
 }
 
+static int pv_mount_install(struct pv_init *this)
+{
+	char path[PATH_MAX];
+	struct stat st;
+	sprintf(path, "/lib/pv/pantavisor-installer");
```

Comment:
setting this in paths.h would be the way to go as "/lib" mount point is configurable

Reply:
yeah i just moved this patch here... think this will be re-designed as part of firstboot install mode...